src/main/java/com/epam/reportportal/jbehave/JBehaveUtils.java

```diff
@@ -99,7 +102,7 @@ public Launch get() {
 				skippedIssueAttribute.setSystem(true);
 				rq.getAttributes().add(skippedIssueAttribute);
 			}
-
+			rq.getAttributes().addAll(SystemAttributesExtractor.extract(AGENT_PROPERTIES_FILE));
```

Comment:
New functionality should be covered with unit-tests despite the fact the there were no unit-tests before.